data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/stream/ShardConsumer.java

```diff
@@ -242,100 +241,88 @@ public void run() {
         // Check should skip processing or not.
         if (shouldSkip()) {
             shardProgress.increment();
-            if (acknowledgementSet != null) {
```

Comment:
Will the partition still get marked as COMPLETED with this change? I think it may not be completed since we didn't create any acknowledgment sets for it.